src/Resources/MSGraph.Autorest/README.md

```diff
@@ -136,10 +137,43 @@ directive:
     remove: true
 
   - where:
-      subject: ^application$|^group$|^serviceprincipal$|^user$
-      variant: ^Update$
+      subject: ^application$|^group$|^serviceprincipal$|^user$|^applicationfederatedidentitycredentials$
+      variant: ^Update$|^Create$
     remove: true
 
+  - where:
+      subject: ^applicationfederatedidentitycredentials$
+      parameter-name: applicationid
+    set:
+      parameter-name: ApplicationObjectId
+
+  - where:
+      subject: ^applicationfederatedidentitycredentials$
+      parameter-name: FederatedIdentityCredentialId
+    set:
+      parameter-name: Id
+
+  - where:
+      subject: ^applicationfederatedidentitycredentials$
+      verb: ^Update$
+      parameter-name: Name
+    hide: true
+
+  - where:
+      subject: ^applicationfederatedidentitycredentials$
+      verb: Get|New
+    hide: true
+
+  - where:
+      subject: ^applicationfederatedidentitycredentials$
+    set: 
+      subject: applicationfederatedidentitycredential
```

Comment:
Can we change cmdlet name into {}-AzAdAppFederatedIdentityCredential?

Reply:
> Can we change cmdlet name into {}-AzAdAppFederatedIdentityCredential?

shall I change these names in next release and link the current names as alias?

Reply:
If possible, I hope we can use short name.